package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "hangul-romanization",
+  "package": "Tyriar/hangul-romanization",
```

Comment:
Isn't it better to use repository like this? https://github.com/xtermjs/xterm.js/blob/d6a45b511ecd30c841133381e355ffce483495a1/package.json#L8

Reply:
Either works, but I changed to use the full url :)